src/components/routes/homepage/SearchBar.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  Container,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  makeStyles
+} from '@material-ui/core';
+import { Search as SearchIcon } from '@material-ui/icons';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    padding: theme.spacing(2),
+    width: '100%',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(2, 1fr)',
+    gridGap: theme.spacing(1),
+    alignItems: 'center'
+  },
+  searchBar: {
+    width: 'min(100%, 300px)',
+    color: theme.palette.primary.main
+  },
+  sortParamSelector: {
+    width: 'min(100%, 150px)',
+    justifySelf: 'end'
+  }
+}));
+
+export const SearchBar = ({ setFilter, sortParam, setSortParam }) => {
+  const classes = useStyles();
+
+  const inputLabel = useRef(null);
+  const [labelWidth, setLabelWidth] = useState(0);
+
+  useEffect(() => {
+    setLabelWidth(inputLabel.current.offsetWidth);
+  }, []);
+
+  return (
+    <Container className={classes.container}>
+      <OutlinedInput
+        className={classes.searchBar}
+        placeholder="Search event(s)"
```

Comment:
Just use `"Search"`, better presentation.

Reply:
> I see that `sortParam` is actually not being used in `filterEvents()` yet, I take it that we still have to figure out the implementation for `subscribers` for both frontend and backend?

yess that's true, my plan after this being merged is to start writing the query and mutation on phoenix. But i kinda think we need to discuss the api design for that, what do you think?